src/components/AppBreadcrumbs/AppBreadcrumbs.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Breadcrumbs, Link } from '@material-ui/core';
+import { Link as RouterLink } from 'react-router-dom';
+
+function capitalize(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+const styles = (theme: any) => ({
+  breadcrumb: {
+    fontStyle: 'italic',
+  },
+  terminalBreadcrumb: {
+    fontStyle: 'italic',
+    color: theme.palette.primary.dark,
+    cursor: 'default',
+    '&:hover': {
+      textDecoration: 'none',
+    },
```

Comment:
You replace all of this with (also for consistency):
```suggestion
    ...theme.mixins.jadeLink,
```

Reply:
This would actually achieve the opposite of what we want. The terminal breadcrumb should be styled differently than the clickable breadcrumbs before it.